### PR TITLE
[ADD] new feature to view queue jobs from program & cycle

### PR DESCRIPTION
**Pull request by nhatnm0612**
## What this PR does?

Add a new smart button in button box for cycle and program to links them to related queue jobs

image
image

### CODE REVIEW

**Review by jeremi — CHANGES_REQUESTED** [review]
Can we have a specific access right for this?

**Comment by nhatnm0612** [comment]
> Can we have a specific access right for this?

sure

File: g2p_programs/views/programs_view.xml [hunk 1]
```diff
@@ -197,6 +208,7 @@ Part of OpenG2P. See LICENSE file for full copyright and licensing details.
                                         widget="badge"
                                         optional="show"
                                     />
+                                    <field name="entitlement_ids" widget="many2many_tags" invisible="1" />
```
Comment (gonzalesedwin1123) [inline g2p_programs/views/programs_view.xml]: This was already replaced with `entitlements_count` field in the latest merge due to performance issue. Considering that this fixed is to resolve an issue you encountered with spp_dashboard, which refers to the entitlement_ids in xpath to insert a button, the spp_dashboard was also fixed in a separate approved PR.  I suggest that while that PR is not yet merged you uninstall the spp_dashboard in your odoo instance.

**Review by gonzalesedwin1123 — CHANGES_REQUESTED** [review]

File: g2p_programs/views/programs_view.xml (hunk 1 above)
Comment (nhatnm0612) [reply g2p_programs/views/programs_view.xml]: @gonzalesedwin1123 okay, I will drop this commit

**Review by gonzalesedwin1123 — APPROVED** [review]

**Pull request merged**